### DISCUSSION TITLE
Adding `.editorconfig` file

### DISCUSSION THREAD

**Discussion by agung2001** [tailpress/tailpress#163] [Ideas]
hi @jeffreyvr 👋😄

I'd like to add `.editorconfig` file to the project
```
root = true

[*]
charset = utf-8
end_of_line = lf
insert_final_newline = true
trim_trailing_whitespace = true
indent_style = tab

[*.json]
indent_style = space
indent_size = 2
```

I think this file is important for maintaining a consistent coding style for the developer.
[EditorConfig](https://editorconfig.org/) is also supported by a wide range of IDEs now.
If that's okay for you, I'm gonna create a pull request (PR) containing this file, so what do u think?

**Comment by jeffreyvr** [comment]
Hi! I like the idea but need to think about this. Coding standards may vary per developer/team. Some may use PSR, others WordPress Coding Standards etc..

So adding this might be annoying to some users as it presets standards they would maybe not use.

**Comment by agung2001** [comment]
aaa you're right, closing the discussion 😁👍
